web/pom.xml

```diff
@@ -65,6 +65,12 @@
             <artifactId>lombok</artifactId>
             <version>1.18.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.alibaba</groupId>
```

Comment:
Do we have other fastjson alternative to use?

Reply:
I think we could remove that, because it is just for testing